### DISCUSSION TITLE
This custom filetype syntax highlighting is not working

### DISCUSSION THREAD

**Discussion by ghost** [geany/geany#3804] [Q&A] [answered]
Go to `bacon_trunk/syntax/geany`:

https://chiselapp.com/user/bacon/repository/bacon/dir?ci=tip

**Comment by elextr** [comment, marked as answer]
Well, no lexer to define the syntax is set, so no wonder its not working ;-)

https://www.geany.org/manual/current/index.html#custom-filetypes
